package-build.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config
+              - config-release
+          - kbld:
+              paths:
+              - '-'
+      export:
+      - imgpkgBundle:
+          image: ghcr.io/vmware-tanzu/carvel-kapp-controller
```

Comment:
Just as a reminder on this PR --> We ought to maintain our current artifact's as well as provide the bundle. So it should probably be named similar to secretgen-controller's bundle: https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/package-build.yml#L25

Reply:
Thanks @neil-hickey for pointing this out. Changed the URL to `ghcr.io/vmware-tanzu/carvel-kapp-controller-package-bundle`